api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -577,6 +577,23 @@ def test_add_contributors_errors(
             expect_errors=True)
         assert res.status_code == 409
 
+    #   test_unregistered_contributor_invalid_email(
+        data = {
+            'data': {
+                'type': 'contributors',
+                'attributes': {
+                    'permission': 'admin',
+                    'email': 'jdoe@example.com',
+                    'full_name': 'John Doe'
+                }
+            }
+        }
+        res = app.post_json_api(
+            url_public, data, auth=user.auth,
+            expect_errors=True)
+        assert res.status_code == 400
```

Comment:
👍 NodeContributorsCreateSerializer catches the ValidationError thrown on the model and passes along the error.